app/utils/formSpyUtils.tsx

```diff
@@ -25,13 +25,21 @@ export const spyFormError = (render: (error: any) => ReactNode) => (
   </FormSpy>
 );
 
-export const spySubmittable = (render: (submittable: boolean) => ReactNode) => (
+type SpySubmittableOptions = {
+  allowPristine?: boolean;
+};
+export const spySubmittable = (
+  render: (submittable: boolean) => ReactNode,
+  { allowPristine = false }: SpySubmittableOptions = {}
```

Comment:
It allows you to submit the form even if no values are changed. We want this in forms because you may want to simply use the template, which is provided in default values.